cds/config.py

```diff
@@ -38,6 +38,7 @@
 from invenio_records_rest.facets import range_filter, terms_filter
 
 from .modules.deposit.facets import created_by_me_aggs
+from .modules.migrator.cli import dumps  # noqa
```

Comment:
I did the same as in [invenio-indexer](https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/ext.py#L34). :smile: 
The main `indexer` command is registered on `invenio-search` and `invenio-indexer` is extending it with more subcommands.
I need only to "load" the file to automatically register subcommands for [dumps](https://github.com/inveniosoftware/invenio-migrator/blob/master/invenio_migrator/cli.py#L40).